### ISSUE TITLE
AppInviteDialogFunction not working on android

### ISSUE THREAD

**Issue by mali90** [nodrock/ANE-Facebook#8]
Thanks for this great extension.

The appInviteDialogue function is not opening the dialogue on android. The rest of the features are working perfectly. Am I supposed to add some activity in the android manifestAdditions for the invite dialogue?

When I call the function, the following trace happens but no dialogue is opened:
[AirFacebook][NATIVE] AppInviteDialogFunction

The invite dialogue is working on iOS.

**Comment by umeshbeecavegames** [comment]
Yes, add the activity in manifest.  It will work.

**Comment by mali90** [comment]
Can you tell me which activity?

**Comment by mali90** [comment]
I think I've found the problem. It's not the extension. I had the Google Play Package Name in the Facebook developer console wrong and I haven't published the app yet. Hopefully it will work after that.

Thanks.

**Comment by mali90** [comment]
It's still not working.

I have published the app on play store. Fixed the package name in the facebook app developer console. added all the activities in the xml file.

I can log in successfully and receive the access token. The following is logged when I call the appInviteDialog function:

[AirFacebook][NATIVE] AppInviteDialogFunction

But nothing happens after that. No dialog is opened. 

Any suggestions on how to fix this issue?

Thanks

**Comment by nodrock** [comment]
Did you set appLinkUrl properly? Do you see any other error or warning in LogCat?

**Comment by mali90** [comment]
I gave the url of my app website. The dialog is opening on iOS with the url.  Is there a specific structure of the appLinkUrl to be followed in case of android? No other errors or warnings are shown.

**Comment by nodrock** [comment]
Could you please try to add this activity to your application.xml file?

``` xml
<activity android:name="com.freshplanet.ane.AirFacebook.AppInviteActivity" android:theme="@android:style/Theme.Translucent.NoTitleBar"
                    android:configChanges="keyboard|keyboardHidden|screenLayout|screenSize|orientation" />
```

I think I forgot to add this activity to README. I am very sorry. Sometimes maintaining the documentation is harder than writing code.

**Comment by mali90** [comment]
Thanks for the quick reply.

I have added the activity but now the app crashes after appInviteDialog function is called. The log still only shows:
[AirFacebook][NATIVE] AppInviteDialogFunction

I am using the latest release of your ane.

**Comment by j3k0** [comment]
@nodrock Right! Writing doc for new features is as important as writing code, that's what we (the lazy users) will look at! Keep up the good work, I'll need this appInviteDialog as well, so I'm following.

**Comment by nodrock** [comment]
I know exactly where is the problem. Facebook Android SDK is missing CREATOR for parcelable class AppInviteContent. I am working of fix and release it today. Thank you guys for using this extension.

**Comment by nodrock** [comment]
Fixed in release 4.5.0. Thank you for patience.
